test/math/core.spec.js

```diff
@@ -1,11 +1,76 @@
-import ava from 'ava'
+import ava from 'ava';
 
 import {
+    isWithin,
+    isWithinEpsilon,
     calculateMiddle,
     clamp,
     generateRandomOctalWithLength
 } from '../../src/assets/scripts/client/math/core';
 
+ava('.isWithin() returns true if value is within (inclusive) two given values', (t) => {
+    // we want to test the function opperates correctly in the negative
+    // range, the positive range, as well as right in the middle.
+    const tests = [0, 1, -1];
+
+    // we will test providing the 'limit1' and 'limit2' both
+    // ways since the function is supposed to work that way
+    for (const number of tests) {
```

Comment:
😌 so much more readable, thank you!

Reply:
Not sure but I think your are mixing tests for `isWithin()` and `isWithinEpsilon()`. A probably contributing factor is that I used EPSILON in the test of `isWithin()` too.

Anyway since you like it better I rewrote the test for `isWithinEpsilon()` in the same style. And I agree a hundred percent it's way clearer now.